src/lib/v2/mappers/local/sidebarTree.tsx

```diff
@@ -86,13 +86,16 @@ function getWorkspaceChildrenOrdered(
   sortingOrder: SidebarTreeSortingOrder,
   workspaceRows: LocalTreeItem[]
 ): LocalTreeItem[] {
+  const compare = (a: LocalTreeItem, b: LocalTreeItem) => {
+    return ~~b.id.startsWith('folder') - ~~a.id.startsWith('folder')
```

Comment:
Can you please replace hard-coded constant with a predefined value we have: `FOLDER_ID_PREFIX`
It can be found in `src/lib/db/consts.ts`